cobc/cobc.c

```diff
@@ -8217,7 +8217,7 @@ process_compile (struct filename *fn)
 		"%s /c %s %s /Od /MDd /Zi /FR /c /Fa\"%s\" /Fo\"%s\" \"%s\"" :
 		"%s /c %s %s     /MD          /c /Fa\"%s\" /Fo\"%s\" \"%s\"",
 			cobc_cc, cobc_cflags, cobc_include, name,
-			name, fn->translate);
```

Comment:
check if that's reasonable my manually doing:

```cmd
cobc -v -S hello.cob
cobc -v hello.asm
cobcrun hello
```

Reply:
Well, since MSVC's assembly listing is "informational", we're not expecting this to work, do we ?
Even calling ml64.exe directly on the .asm file, I get some errors.